app/controllers/admin/adventures_controller.rb

```diff
@@ -2,6 +2,7 @@ class Admin::AdventuresController < Admin::BaseController
   load_permissions_and_authorize_resource
 
   def index
+    set_adventures
```

Comment:
If I remove it, when clicking at Adventures from the Introduction show view all adventures come up (from other Introductions as well), which I don't think is desired.

Not sure though if it can be done in a more elegant way, probably..., but I believe I copied the implementation of groups from the same view (Introduction#show)